### PR TITLE
Telegram.TelegramDesktop 4.1.0

### PR DESCRIPTION
**Pull request by ottnorml**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/70071)

### CODE REVIEW

File: manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 4.1.0
```
Comment (ItzLevvie) [inline manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.installer.yaml:5]:
```suggestion
PackageVersion: 4.1
```
This must match Control Panel.

You'll also need to change `manifests/t/Telegram/TelegramDesktop/4.1.0` to `manifests/t/Telegram/TelegramDesktop/4.1`

File: manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 4.1.0
```
Comment (ItzLevvie) [inline manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.1
```
This must match Control Panel.

You'll also need to change `manifests/t/Telegram/TelegramDesktop/4.1.0` to `manifests/t/Telegram/TelegramDesktop/4.1`

File: manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 4.1.0
```
Comment (ItzLevvie) [inline manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.yaml:5]:
```suggestion
PackageVersion: 4.1
```
This must match Control Panel.

You'll also need to change `manifests/t/Telegram/TelegramDesktop/4.1.0` to `manifests/t/Telegram/TelegramDesktop/4.1`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.installer.yaml (hunk 1 above)
Comment (ottnorml) [reply manifests/t/Telegram/TelegramDesktop/4.1.0/Telegram.TelegramDesktop.installer.yaml]: Thx for your feedback. I applied the changes.

**Comment by ottnorml** [comment]
I close this PR in favor of #70336.

**Pull request closed without merging**